apps/web/components/eventtype/EventAppsTab.tsx

```diff
@@ -106,9 +108,21 @@ export const EventAppsTab = ({ eventType }: { eventType: EventType }) => {
       {!shouldLockDisableProps("apps").disabled && (
         <div className="bg-muted rounded-md p-8">
           {!isLoading && notInstalledApps?.length ? (
-            <h2 className="text-emphasis text-lg font-semibold">{t("available_apps")}</h2>
+            <>
+              <h2 className="text-emphasis mb-2 text-xl font-semibold leading-5 tracking-[0.01em]">
+                {t("available_apps_lower_case")}
+              </h2>
+              <p className="mb-6 text-sm font-normal text-gray-600">
+                <Trans i18nKey="available_apps_desc">
+                  You have no apps installed. View popular apps below and explore more in our &nbsp;
+                  <Link className="cursor-pointer underline" href="/apps">
+                    App Store
+                  </Link>
+                </Trans>
+              </p>
+            </>
           ) : null}
-          <div className="before:border-0">
+          <div className="bg-default divide-y rounded-md border before:border-0">
```

Comment:
<img width="879" alt="Screenshot 2023-06-19 at 3 16 16 PM" src="https://github.com/calcom/cal.com/assets/53316345/2cad0db8-2d83-4f0e-b76c-cbebd436fef6">

border color should be same as you see around Qr code app in screenshot